my_game.py

```diff
@@ -29,6 +31,39 @@
 FIRE_KEY = arcade.key.SPACE
 
 
+def load_tmx(tmx_file, self):
+
+    layer_names = [
+        "background",
+        "impassable",
+        "objects-impassable",
+        "objects-passable",
+        "pressure-plates"
+    ]
+    # Layer names and use of "special_hash".
+    layer_options = {
+        layer_names[0]: {"use_spatial_hash": True},
+        layer_names[1]: {"use_spatial_hash": True},
+        layer_names[2]: {"use_spatial_hash": True},
+        layer_names[3]: {"use_spatial_hash": True},
+        layer_names[4]: {"use_spatial_hash": True}
+    }
+
+    # Loads the tile-map, layer_options, and how big the tmx is.
+    tile_map = arcade.load_tilemap(
+        tmx_file, layer_options=layer_options, scaling=TILE_SCALING
+    )
+
+    self.dict_list = {
```

Comment:
No need for self (this is not a class/object)

Reply:
Changed! :]